docs/querying/api.md

```diff
@@ -1294,3 +1294,16 @@ Enable the remote write receiver by setting
 endpoint is `/api/v1/write`. Find more details [here](../storage.md#overview).
 
 *New in v2.33*
+
+## OTLP Receiver
+
+Prometheus can be configured as a receiver for the OTLP Metrics protocol. This 
+is not considered an efficient way of ingesting samples. Use it
+with caution for specific low-volume use cases. It is not yet suitable for
+replacing the ingestion via scraping yet.
+
+Enable the OTLP receiver by the feature flag
+`--enable-feature=otlp-write-receiver`. When enabled, the OTLP receiver
+endpoint is `/otlp/v1/metrics`.
```

Comment:
hi, the endpoint maybe `/api/v1/otlp/v1/metrics`.

Reply:
you are right. 
code example

```go
otlphttp, err := otlpmetrichttp.New(
		ctx,
		otlpmetrichttp.WithInsecure(),
		otlpmetrichttp.WithEndpoint("localhost:9090"),
		otlpmetrichttp.WithURLPath("/api/v1/otlp/v1/metrics"),
)
```